server/cmwell-data-tools/src/main/scala/cmwell/tools/data/utils/akka/Retry.scala

```diff
@@ -177,13 +177,20 @@ object Retry extends DataToolsLogging with DataToolsConfig {
           }
           else {
             // server error
-            // special case: sparql-processor //todo: remove this in future
+            // special case: sparql-processor //TODO: remove this in future
 
             e.dataBytes.runFold(ByteString.empty)(_ ++ _).map(_.utf8String).map{ entity=>
               logger.warn(s"[$errorID] server error. Body of response: $entity, headers: ${headersString(h)}")
             }
 
-            if (e.toString contains "Fetching") {
+            // special case: sparql-processor //TODO: remove this in future
```

Comment:
this is a Client error (400) and should be in a different case (line 231)

Reply:
Yes, it retries requests to SpHandler over and over which returns 400 Parsing Error.